text-dream/public/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
all html/js files should have copyright headers

Reply:
Done.